.env

```diff
@@ -0,0 +1,4 @@
+# Set in order to run specific versions of HA & Frigate.
+#
+# HA_VERSION=stable
+# FRIGATE_VERSION=stable-amd64
```

Comment:
Ideally this file should be in .gitignore, otherwise people would have to exclude it when doing their commits.

You may rename this one as

.env.example

Perhaps

Reply:
SGTM.